core/service/src/components.rs

```diff
@@ -303,17 +322,18 @@ pub trait Components: Sized + 'static {
 	type Backend: 'static + client::backend::Backend<FactoryBlock<Self::Factory>, Blake2Hasher>;
 	/// Client executor.
 	type Executor: 'static + client::CallExecutor<FactoryBlock<Self::Factory>, Blake2Hasher> + Send + Sync + Clone;
-	/// Extrinsic pool type.
-	type TransactionPoolApi: 'static + txpool::ChainApi<
-		Hash = <<Self::Factory as ServiceFactory>::Block as BlockT>::Hash,
-		Block = FactoryBlock<Self::Factory>
-	>;
 	/// The type that implements the runtime API.
 	type RuntimeApi: Send + Sync;
 	/// A type that can start the RPC.
 	type RPC: StartRPC<Self>;
-	/// A type that can create the network params.
-	type CreateNetworkParams: CreateNetworkParams<Self>;
+	// TODO [ToDr] Tratify transaction pool and allow people to implement their own.
```

Comment:
Please create an issue in github and put a link into the comment.

Reply:
Addressed and tests fixed.